src/ui/ToolBar.js

```diff
@@ -19,6 +21,13 @@ export default class ToolBar extends React.Component {
               >
                 End process
               </button>
+              <button
+                className="btn btn-default"
+                disabled={this.props.disableDebug}
+                onClick={this.props.onDebugClick}
+              >
+                Debug process
```

Comment:
To be consistent with Electron's wording, I'd rather call this "Open Dev Tool". The naming of methodes, event name and so on should BTW match this.